nservicebus/rabbitmq/delayed-delivery.md

```diff
@@ -0,0 +1,129 @@
+exchangeN -->|bit=0| exchangeN-1
+queueN -->|expired<br>message| exchangeN-1
+
+classDef exchangeClass stroke:#000000,stroke-width:2px;
+class exchangeN,exchangeN-1 exchangeClass
+
+end
+
+```
+
+The levels are connected in this manner, from highest (27) to lowest (0). Each level's routing key's add wildcards as needed so that they are looking at the portion of the message's routing key that corresponds to its level.
+
+
+### Delivery
+
+The final delay-level exchange is bound to the delivery exchange instead of another level. Every endpoint that can receive a delayed message will have a binding to this exchange with a routing key corresponding to the endpoint's name, so the message will be delivered to the endpoint's queue.
+
+
+### Example
+
+Using a simplified version of the delay infrastructure that has 4 levels (0-3), here is an example of sending a message with a delay of 5 seconds to an endpoint called `destination`:
+
+1. The message is published to the level 3 exchange with the following routing key: `0.1.0.1.destination`
+1. The level 3 bit of the routing key is `0`, so the message is routed to the level 2 exchange. (**0**.1.0.1.destination)
+1. The level 2 bit of the routing key is `1`, so the message is delivered to the level 2 queue. (0.**1**.0.1.destination)
+1. After 4 seconds, the message expires and is routed to the level 1 exchange.
+1. The level 1 bit of the routing key is `0`, so the message is routed to the level 0 exchange. (0.1.**0**.1.destination)
+1. The level 0 bit of the routing key is `1`, so th message is delivered to the level 0 queue. (0.1.0.**1**.destination)
+1. After 1 second, the message expires and is routed to the delivery exchange.
+1. The final portion of routing key is `destination`, so the message is delivered to the endpoint's queue. (0.1.0.1.**destination**)
+
+```mermaid
+graph LR
+
+subgraph Example: delay of 5 seconds
+
+exchange3(Level 3)
+exchange2(Level 2)
+exchange1(Level 1)
+exchange0(Level 0)
+exchange-delivery(delay-delivery)
+q3[fa:fa-hourglass-half 8sec]
+q2[fa:fa-hourglass-half 4sec]
+q1[fa:fa-hourglass-half 2sec]
+q0[fa:fa-hourglass-half 1sec]
+
+exchange3 .-> q3
+exchange3 ==>exchange2
+q3 .-> exchange2
+
+exchange2 ==> q2
+exchange2 .->exchange1
+q2 ==> exchange1
+
+exchange1 .-> q1
+exchange1 ==>exchange0
+q1 .-> exchange0
+
+exchange0 ==> q0
+exchange0 .->exchange-delivery
+q0 ==> exchange-delivery
+
+exchange-delivery ==> dest[destination]
+
+classDef exchangeClass stroke:#000000,stroke-width:2px;
+class exchange3,exchange2,exchange1,exchange0,exchange-delivery exchangeClass
+
+classDef usedQueue fill:#11ff00;
+class q0,q2 usedQueue
+
+end
+```
+
+
+## Settings
+
+
+### Disable the timeout manager
+
+By default, the timeout manager continues to run to allow any preexisting delayed messages stored in the persistence database to be successfully sent. However, once the persistence database is empty, there is no more need for the timeout manager. It can be disabled by calling:
```

Comment:
We have Particular/NServiceBus.RabbitMQ/issues/349 to track considering making changes to the default values in the future, but since we haven't actually committed to making a change here, I'd prefer not to mention it in the docs yet.